lnwallet/channel_test.go

```diff
@@ -4824,24 +4846,53 @@ func TestChanAvailableBalanceNearHtlcFee(t *testing.T) {
 	// nonDustHtlc+commitfee+aliceReserve+htlcfee. The available balance
 	// reported will just be nonDustHtlc, since the rest of the balance is
 	// reserved.
-	expBalance = aliceNonDustHtlc
-	checkBalance(t, expBalance)
+	expAliceBalance = aliceNonDustHtlc
+	expBobBalance = bobBalance - bobReserve
+	checkBalance(t, expAliceBalance, expBobBalance)
```

Comment:
These expectations refer to variables that are also updated themselves. I think it is more robust to assert against literal values here, to be sure that no mistake in the test code itself can cause the wrong thing being asserted.

Reply:
Don't see how that would be better

Reply:
It is more difficult to see that this test asserts the right behavior.